torch/testing/_internal/common_methods_invocations.py

```diff
@@ -978,7 +978,7 @@ def method_tests():
         ('mm', (S, M), ((M, S),), '', (True,)),
         ('bmm', (M, S, M), ((M, M, S),), '', (True,)),
         ('mv', (S, M), ((M,),), '', (True,)),
-        ('ger', (S,), ((M,),)),
+        ('outer', (S,), ((M,),)),
```

Comment:
This is a pretty innocuous change, but it would be nice if adding an aliased schema test didn't fail. Since the change here is not too complicated do you mind doing it as part of this PR? We can expose the https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/normalize_ops.cpp#L10 map here in an API and try to normalize the op name if we didnt fine any nodes with the original name

Reply:
I didn't know that map existed. That's pretty sweet! Will use it.